Coastal Flooding Implementation

jsnrdl opened the pull request:
**ccai/climate/coastal.py**

in function fetch coastal:
Coastal flooding is now showing on the front end as part of the JSON RESPONSE

vict0rsch commented on ccai/config.py:
```diff
@@ -31,8 +31,8 @@ class ConfigSingleton(Singleton):
 
     SECRET_KEY = os.environ.get("SECRET_KEY") or "secret-key"
     API_KEYS_NAME = ["GEO_CODER_API_KEY", "STREET_VIEW_API_KEY"]
-    GEO_CODER_API_KEY = "AIzaSyBDueAV41YFgKvKZ72UtTHQ3PiMuedwZ0k"
```
!! @jsnrdl what is this? How come this is committed?

vict0rsch replied:
API keys should **NEVER** be committed... did you generate new ones?

jsnrdl replied:
that has been modified, the keys have been changed and put in a file.

vict0rsch commented:
what's the state of this

jsnrdl commented:
Following your advice, I changed the keys in the google platform, removed
the hard coded keys in the code, put my new keys in a file, and the file is
flagged in .gitignore.

Got my lesson!

--JA


On Fri, May 22, 2020 at 3:49 PM Victor Schmidt <notifications@github.com>
wrote:

> what's the state of this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cc-ai/floods-backend/pull/37#issuecomment-632886465>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AISNC2XOU5O2BMCIEBROY6DRS3JMJANCNFSM4M2RDHBA>
> .
>

vict0rsch commented:
I meant, why is it not merged @jsnrdl :)
